ci/gpu/build.sh

```diff
@@ -36,7 +36,7 @@ nvidia-smi
 logger "Activate conda env..."
 source activate gdf
 conda install -c rapidsai/label/cuda${CUDA_REL} -c rapidsai-nightly/label/cuda${CUDA_REL} cudf=${CUDF_VERSION} rmm=${RMM_VERSION} nvstrings=${NVSTRINGS_VERSION}
-conda install -c conda-forge lapack cmake==3.14.3 umap-learn
+conda install -c conda-forge lapack cmake==3.14.3 umap-learn cupy
```

Comment:
This is just a build-time/test-time dependency on cupy, right? Not runtime? Works for me..

Reply:
Not even a build dependency since the code has exception handling to manage gracefully and skip tests that require `cupy` :)